recipe/meta.yaml

```
@@ -1,9 +1,10 @@
-{% set version = "7.0.4" %}
-{% set sha256 = "1f0597b91740742beec49ca1ca59e7b71f63483b7495068b4f0f685c0cda9e69" %}
+{% set version = "7.0.5" %}
+{% set sha256 = "faf8dd3bfa4a3b0036e8b62a5f647a0596bfd579f31c6cc00b2ce2f82bcf9de1" %}
 
 package:
   name: epics-base
-  version: {{ version }}
+  # Always use 4 numbers in the version to match run_exports!
+  version: 7.0.5.0
```

I suggest parametrizing the version here. That way we don't have to keep 2 places in sync.
```suggestion
  version: {{ version }}.0
```